### ISSUE TITLE
Establish a translation percentage threshold for languages to be included in releases

### ISSUE THREAD

**Issue by cbjeukendrup** [musescore/MuseScore#16779]
**The problem**
Currently, translations for 68 languages are packaged with MuseScore. However, only a fraction of those is being actively translated. At the moment of writing, only 13 languages are 100% translated, and 16 others have less than 2000 untranslated strings (of the 7845 total strings). 

It's not only a bit misleading to include "translations" that leave half of the app untranslated; in some cases it's just problematic. For example, we have `Arabic (ar)`, which is very reasonably translated with 65%, but we have also `Arabic (Algeria) (ar_DZ)`, which contains exactly zero translated strings. Users from Algeria will get this variant, with no translations at all, instead of the properly translated "generic" Arabic.

This problem also prevents us from accepting new languages: we need to be sure in advance that someone will indeed work dedicatedly on the translations for that new language. 

**The solution**
I propose that we introduce a threshold on the translation percentage for languages to be included in releases. Let's choose 60% to start with. 

This way, translations that are not really translations are excluded until they get updated, and we can freely accept new languages on Transifex, without worrying whether they will be translated enough.

**Comment by Jojo-Schmitz** [comment]
Yes, I'd remove `Arabic (Algeria) (ar_DZ)` and I guess `Arabic (Egypt) (ar_EG)` too.
Propbably some others too, like `Igbo (ig)`, `Kabyle (kab)` and `Sicilian (scn)`, but those at least don't harm.

Only exception from that 60% rule should be `English (US) (en_US)`

**Comment by Jojo-Schmitz** [comment]
We could also 'seed' (or merge)  `Arabic (Algeria) (ar_DZ)` and `Arabic (Egypt) (ar_EG)` with `Arabic (ar)` and so get them to a reasonable state right away.

**Comment by Jojo-Schmitz** [comment]
Also I think we should change `Mongolian (Mongolia) (mn_MN)` to `Mongolian (mn)` and  to `Hindi (hi)`

**Comment by FrancRos31** [comment]
> Yes, I'd remove `Arabic (Algeria) (ar_DZ)` and I guess `Arabic (Egypt) (ar_EG)` too. Propbably some others too, like `Igbo (ig)`, `Kabyle (kab)` and `Sicilian (scn)`, but those at least don't harm.
> 

I see Sicilian, interesting. I know this thread is a year old, but, just for curiosity, were expected other translations of MuseScore in other Italian dialects?
